Automattic/jetpack
# Improve semantics of form checkboxes for assistive technology

Fixes #

#### Changes proposed in this Pull Request:

* Puts the checkboxes in a list for assistive technology
* Adds a header above the list for assistive technology

#### Testing instructions:



* The checkboxes in the subscription form should be in a list
* That list should have a header which labels the list.
* When viewing the subscription options underneath the comments box on a site, the check boxes should not see bullets.
![image](https://user-images.githubusercontent.com/44990/43659234-37cad834-9710-11e8-83fd-7b3661bf927d.png)


#### Proposed changelog entry for your changes:

Improves the semantic markup of the checkboxes to subscribe to a comment,
